modules/restore-add-master.adoc

```diff
@@ -9,9 +9,10 @@ Follow these steps to add a master host back to the etcd cluster. This procedure
 
 .Prerequisites
 
-* Access to the cluster as a user with the `cluster-admin` role.
-* SSH access to the master host to add to the etcd cluster.
-* The IP address of an existing active etcd member.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have SSH access to the master host to add to the etcd cluster.
+* You have the IP address of an existing active etcd member.
```

Comment:
You have SSH access to the one of the healthy master host.

Reply:
@retroflexer Was this comment supposed to be for the new regenerating procedure's prereq list?

Reply:
Looking at the steps in that procedure, I assume they need both access to a healthy master and to the new one, correct?

Reply:
Yes.